etc/leapp/files/devel-livemode.ini

```diff
@@ -0,0 +1,7 @@
+# Configuration for the *experimental* livemode feature
+# For the full list of configuration options, see models/livemode.py
```

Comment:
I would add a note that this configuration will be replaced completely in close future - basicaly right after the introduction of leapp actors configurations.